stable/spark/values.yaml

```diff
@@ -8,31 +8,37 @@ Spark:
 
 Master:
   Name: master
-  Image: "k8s.gcr.io/spark"
-  ImageTag: "1.5.1_v3"
+  Image: "dbanda/spark2.4"
+  ImageTag: "v3"
   Replicas: 1
   Component: "spark-master"
   Cpu: "100m"
-  Memory: "512Mi"
+  Memory: "1024Mi"
   ServicePort: 7077
   ContainerPort: 7077
   # Set Master JVM memory. Default 1g
   # DaemonMemory: 1g
   ServiceType: LoadBalancer
 
+WorkerLB:
+  Name: workerlb
+  Component: "workerlb"
+  ServicePort: 8080
+  ContainerPort: 8080
+
 WebUi:
   Name: webui
   ServicePort: 8080
   ContainerPort: 8080
 
 Worker:
   Name: worker
-  Image: "k8s.gcr.io/spark"
-  ImageTag: "1.5.1_v3"
-  Replicas: 3
+  Image: "dbanda/spark2.4"
```

Comment:
Why are we changing the container image source?

Reply:
The containers that existed are very outdated and did not provide dockerfiles (only bits) so we decided to swap them out with something newer and more open.